tests/unit/options/unit_tests_sender_option_parser.rb

```diff
@@ -358,6 +371,39 @@ def test_sender_option_parser_user_msg_content_from_file_long
     )
   end # test_sender_option_parser_user_msg_content_from_file_long
 
+  def test_sender_option_parser_user_content_type_msg_content_from_file_values
+    for type, param, expect in [%w(string 1 1), ["int", "1", 1], ["long", "1", 1],
+                                ["float", "1", 1.0], ["bool", "true", true]]
+      file = Tempfile.new("test_sender_option_parser_user_content_type_msg_content_from_file_values")
+      begin
+        file.write(param)
+        file.flush
+        sender_options_user_msg_content_map_item_long = \
+          Options::SenderOptionParser.new([
+            "--msg-content-from-file", file.path, "--content-type", type
+        ])
+        assert_equal(
+          expect,
+          sender_options_user_msg_content_map_item_long.options.msg_content
+        )
+      rescue
```

Comment:
Not sure if this should be rescue or ensure.

Reply:
`ensure`, will fix

Reply:
done